modules/insights-operator-configuring.adoc

```diff
@@ -31,15 +56,45 @@ Red Hat recommends you consult Red Hat Support before making changes to the defa
 
 .Prerequisites
 
+* Remote health reporting is enabled, which is the default.
 * You are logged in to the {product-title} web console as a user with `cluster-admin` role.
 
 .Procedure
 
-. Navigate to *Workloads* -> *Secrets*.
-. On the *Secrets* page, select *All Projects* from the *Project* list, and then set *Show default projects* to on.
-. Select the *openshift-config* project from the *Project* list.
-. Search for the *support* secret using the *Search by name* field. If it does not exist, click *Create* -> *Key/value secret* to create it.
-. Click the *Options* menu {kebab} for the secret, and then click *Edit Secret*.
-. Click *Add Key/Value*.
-. Enter an attribute name with an appropriate value (see table above), and click *Save*.
-. Repeat the above steps for any additional configurations.
+. Go to *Workloads* -> *ConfigMaps* and select *Project: openshift-insights*.
+. Click *Create ConfigMap*.
+. Select *Configure via: YAML view* and enter your configuration preferences, for example
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: insights-config
+  namespace: openshift-insights
+data:
+  config.yaml: |
+    dataReporting:
+      obfuscation:
+        - networking
+        - workload_names
+    sca:
+      disable: false
+      interval: 2h
+    alerting:
+       disabled: false
+binaryData: {}
+immutable: false
+----
+
+. Optionally, select *Form view* and enter the necessary information that way.
+. In the *ConfigMap Name* field, enter *insights-config*.
+. In the *Key* field, enter *config.yaml*.
+. For the *Value* field, either browse for a file to drag and drop into the field or enter your configuration parameters manually.
++
+[IMPORTANT]
+====
+Be careful to indent each parameter two spaces under its parent.
```

Comment:
I'm not sure this is necessary. I'd ask engr. that is just a standard rule of thumb for YAML.